bin/server.js

```diff
@@ -0,0 +1,23 @@
+const { createServer } = require('http');
+const { parse } = require('url');
+const fs = require('fs');
+
+const next = require('next');
+
+const dev = process.env.NODE_ENV !== 'production';
+const app = next({ dev });
+const handle = app.getRequestHandler();
+
+app.prepare().then(() => {
```

Comment:
Just wondering why we need a `server.js` at all. Doesn't this come baked in with next? I've only ever run anything locally with next, so I don't really know what's needed for a remote app.

Reply:
This is needed because I'm running nginx in front of Node. It does two things, it sets the next server to run on a unix socket and drops a file when it's ready for traffic which is a convention of the nginx buildpack on Heroku.